analyzer/tests/unit/test_checker_handling.py

```diff
@@ -264,6 +264,18 @@ def f(checks, checkers):
         self.assertTrue(all_with_status(CheckerState.ENABLED)
                         (cfg_handler.checks(), low_severity))
 
+        # Enable checkers with a checker group prefix.
```

Comment:
this test is enabling one concrete checker alpha.security.ArrayBound and tests that alpha.security.ArrayBoundv2 should not be enabled. It is not testing "checker group prefix"

Could you please add another test case which enables "alpha.security" group. Then alpha.security.ArrayBound and also alpha.security.ArrayBoundv2 should be enabled.